internal/http/handler.go

```diff
@@ -120,6 +121,41 @@ func HandleVersion(version string) http.HandlerFunc {
 	return func(w http.ResponseWriter, r *http.Request) { fmt.Fprintf(w, `{"version":"%s"}`, version) }
 }
 
+// HandleStatus returns a handler function that returns status
+// information, like server version and server up-time, as JSON
+// object to the client.
+//
+// The handler function should not return any internal data - like
+// keystore backend, cache expiry, etc.
+func HandleStatus(version string, certificate *Certificate) http.HandlerFunc {
+	type Status struct {
+		Version string        `json:"version"`
+		UpTime  time.Duration `json:"uptime"`
+
+		Certificate struct {
+			Expiry   time.Duration `json:"expiry"`
+			DNSNames []string      `json:"dns,omitempty"`
+			IPs      []net.IP      `json:"ip,omitempty"`
+		} `json:"certificate"`
+	}
+	var startTime = time.Now()
+	return func(w http.ResponseWriter, r *http.Request) {
+		w.Header().Set("Content-Type", "application/json")
+		w.WriteHeader(http.StatusOK)
```

Comment:
```suggestion
		w.WriteHeader(http.StatusOK)
```

This is not needed if you are ever going to always send success to client @aead - can we not have provisions for this endpoint to return error at all ?

Reply:
> This is not needed if you are ever going to always send success to client @aead
Correct, not strictly needed when you at least call `Write` once.

> can we not have provisions for this endpoint to return error at all ?
You mean: "KES starts but is not able to perform all operations - like MinIO server starting but some features are not available?"

No, this cannot happen. Either KES starts and is fully functional or KES does not start and fails with an error explaining why.

Reply:
Okay